walkthroughs/eks/base.md

```diff
@@ -0,0 +1,235 @@
+# App Mesh with EKS—Base Deployment
```

Comment:
You may want to update the existing README that has pointers to EKS setup with old instructions.

Reply:
Not entirely sure which README you mean (this: https://github.com/aws/aws-app-mesh-examples/blob/master/README.md maybe?). If so, it currently points to the `examples` directory, yes and I could imagine adding a pointer to the `walkthroughs` directory as well. Unsure about how we want to handle this, maybe @abby-fuller has an opinion on this?

Reply:
Under https://github.com/aws/aws-app-mesh-examples/blob/master/examples/README.md#infrastructure there is a pointer for EKS setup. That can point to this walkthrough.

Reply:
Got it, thanks @kiranmeduri.